src/utils/sdk.ts

```diff
@@ -18,14 +18,23 @@ export const thirdwebClient = createThirdwebClient({
  * Helper functions to handle v4 -> v5 SDK migration.
  */
 
-export const toTransactionStatus = (status: "success" | "reverted"): number =>
+export const fromTransactionStatus = (status: "success" | "reverted") =>
```

Comment:
Renaming this. `transactionStatus` should refer to the v5 shape, so "from" means v5 -> v4 shape and "to" means v4 -> v5 shape.